index.html

```diff
@@ -218,6 +218,7 @@ <h4>Multi-Post Stories</h4>
   
         <section id="about-page" class="about-page-1">
           <div class="about-content">
+            <div class="about-content1">
             <h3 class="about-text">About <br> Myself </h3> 
             <p class="tonic-para">
               Hello I’m a software developer! I can help you build a product ,
```

Comment:
- [ ] Let us fix this, use LinkedIn icon instead of text.
![linkedin](https://user-images.githubusercontent.com/55635977/126771144-3ddec3b0-d02e-43fd-b94c-03f6c8647e9e.png)